source/connect/required-access.txt

```diff
@@ -0,0 +1,97 @@
+.. _required-access:
+
+===============
+Required Access
+===============
+
+.. default-domain:: mongodb
+
+With authentication/authorization enforced on the connected MongoDB
+instance, the user requires appropriate privileges to access various
+features. The following table lists the privileges required to access
+the features as well as the built-in roles that can provide these
+privileges:
+
+.. note::
+   The built-in roles may provide more access than required. You can
+   also create a :ref:`user-defined-roles` to grant specific privileges.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 30 25 20
+
+   * - View
+     - Feature/Action
+     - Privilege(s)
```

Comment:
It almost seems like this should be the title for the current "Feature/Action" column?

Reply:
I think "Privilege(s)" is correct here since we are talking about specific privilege actions. Cc @atsansone

Reply:
Got it